UnityProject/Assets/Scripts/Objects/Medical/CloningConsole.cs

```diff
@@ -223,26 +248,83 @@ public CloningRecord()
 		public void UpdateRecord(LivingHealthMasterBase livingHealth, PlayerScript playerScript)
 		{
 			mobID = livingHealth.mobID;
-			mind = playerScript.Mind;
+			mindID = playerScript.Mind.netId; // Set mindID here
 			name = playerScript.playerName;
 			characterSettings = playerScript.characterSettings;
 			oxyDmg = livingHealth.GetOxyDamage;
 			burnDmg = livingHealth.GetTotalBurnDamage();
 			toxinDmg = livingHealth.GetTotalToxDamage();
 			bruteDmg = livingHealth.GetTotalBruteDamage();
 			uniqueIdentifier = $"{livingHealth.mobID}{playerScript.playerName}".ToHexString();
-			foreach(BodyPart part in livingHealth.SurfaceBodyParts)
-            {
-				BodyPartRecord partRecord = new BodyPartRecord();
+
+			surfaceBodyParts.Clear();
+			foreach (var part in livingHealth.SurfaceBodyParts)
+			{
+				var partRecord = new BodyPartRecord();
 				surfaceBodyParts.Add(partRecord.Copy(part));
-            }
-			foreach(SicknessAffliction sickness in livingHealth.mobSickness.sicknessAfflictions)
-            {
+			}
+
+			sicknessList.Clear();
+			foreach (var sickness in livingHealth.mobSickness.sicknessAfflictions)
 				sicknessList.Add(sickness.Sickness.SicknessName);
-            }
+		}
+
+		// Method to serialize the object to a compact string
+		public string ToCompactString()
+		{
+			var sb = new StringBuilder();
+
+			// Serialize simple fields with delimiters, now includes mindID
+			sb.Append(name).Append("|")
+				.Append(scanID).Append("|")
+				.Append(oxyDmg.ToString("F2")).Append("|")
+				.Append(burnDmg.ToString("F2")).Append("|")
+				.Append(toxinDmg.ToString("F2")).Append("|")
+				.Append(bruteDmg.ToString("F2")).Append("|")
+				.Append(uniqueIdentifier).Append("|")
+				.Append(mobID).Append("|")
+				.Append(mindID).Append("|"); // Include mindID here
+
+			// Serialize surfaceBodyParts
+			foreach (var part in surfaceBodyParts) sb.Append(part.ToCompactString()).Append(";");
+			sb.Append("|");
+
+			// Serialize sicknessList
+			sb.Append(string.Join(",", sicknessList));
+
+			return sb.ToString();
+		}
+
+		// Method to deserialize the object from a compact string
+		public static CloningRecord FromCompactString(string compactString)
```

Comment:
why not use Jsonconvert?

Reply:
This is not performance critical code xD
map loading uses Jsonconvert and it's not the slow bit

Reply:
either way this works just fine and looks good

Reply:
What about if someone edits the paper and has invalid data? also we want to add stuff in the future that is safe such as genetic defects

Reply:
It will be rejected if its invalid.
Either way, they can't directly mess with character sheet stuff because it's stored on the mind.
Also, that can be added by extending the string that gets printed on the paper with the info that we need.

Reply:
But my point is it's easier just to add to a class versus having  add to the class then add theserialise and then do  deserialise steps and if it's conditional God help you xD

Reply:
json doesn't look good imo

Reply:
we could probably turn it into binary, but that would probably take more than the page can hold

Reply:
I think Binary would be good, unless you do some weird stuff, the binary should always be equal to or shorter than your string method

Reply:
the problem is you have to do special stuff for example holding mutations since you can have a variable amount so therefore you have to say how many mutations it has so you can de serialise number,  yyou might as well just use json  works

Reply:
Can't do binary, the result takes more than the page can take.

Reply:
what about json

Reply:
I've said this before two times already, JSON looks ugly and should not be put infront of players.

Reply:
What do you guys mean when you say binary?

Regarding showing this to users, are supposed to be able to read it? Otherwise you could just base64 it

Reply:
Users should be able to read it.